pkgs/development/tools/misc/pwndbg/default.nix

```
@@ -47,6 +50,7 @@ in stdenv.mkDerivation rec {
     makeWrapper ${gdb}/bin/gdb $out/bin/pwndbg \
       --add-flags "-q -x $out/share/pwndbg/gdbinit.py" \
       --prefix PATH : ${binPath} \
+      --set LC_CTYPE "C.UTF-8" \
```

Users might want to set it to something else especially when debugging:
```suggestion
      --set-default LC_CTYPE "C.UTF-8" \
```

---

Mhm. I don't thing we have C.utf-8 on macOS actually.

---

Hmm, yes, on macos we need recompile gdb with custom charset.. :(

---

Hi, Pwndbg maintainer here. We don't really support MacOS well in Pwndbg (hopefully yet), so I think the lack of C.utf-8 on macOS is not an issue here @Mic92.

We should just proceed with merging this and actually further update this to the latest release from yesterday (2023.07.17).